src/AutorestSwift/Models/Complex/ChoiceSchema.swift

```diff
@@ -61,13 +61,13 @@ public class ChoiceSchema: ValueSchema {
 extension ChoiceSchema: SnippetConvertible {
```

Comment:
We can remove this. If you see value in a protocol to annotate which model objects have stencils associated with them, then change it to that. Otherwise we can just remove it. 

```Swift
extension ChoiceSchema: Stencilable {
  func generateTemplate() -> String {
    return renderTemplate(filename: "ChoiceSchema.stencil", dictionary: ["object": self])
  }
}
```

Reply:
Great idea.